app/frontend/js/map.js

```diff
@@ -98,9 +109,20 @@ export function displayMap (sbi, parcels, coordinates) {
   map.addInteraction(selectPointerMove)
 
   selectClick.on('select', function (e) {
-    window.location.href = `/parcel?sbi=${sbi}&sheetId=${e.selected[0].values_.sheet_id}&parcelId=${e.selected[0].values_.parcel_id}`
+    window.location.href = `/parcel?sbi=${sbi}&sheetId=${e.selected[0].values_.sheet_id}&parcelId=${e.selected[0].values_.parcel_id}&mapStyle=${select.value}`
   })
 
+  var select = document.getElementById('layer-select')
```

Comment:
Shouldn't use `var` to declare variables.  Should ideally use `const` or if it must change then `let`.